src/layouts/SideBar/index.tsx

```diff
@@ -31,12 +31,15 @@ function SideBar({ sideBarElement, mainPageElement }: sideBarProps) {
             {sideBarElement}
           </>
         ) : (
-          <Button
-            onClick={handleIsDrawerOpenChange}
-            icon={openCloseIcon}
-            className='top-0 right-0 p-button-rounded p-button-secondary p-button-text absolute'
-            aria-label='Cancel'
-          />
+          <>
+            <Button
+              onClick={handleIsDrawerOpenChange}
+              icon={openCloseIcon}
+              className='top-0 right-0 p-button-rounded p-button-secondary p-button-text absolute'
+              aria-label='Cancel'
+            />
+            {sideBarElement}
```

Comment:
Guess could do some restructure a bit later here. It was intended to leave it out when it is not open. Or possibly could blur this section out rather than showing some cut-off component?

Reply:
I think, it is fine, ya know! Already like the way you did SideBar arrangement there; that's why I was reusing much :D. 

Conceptually, it is just a temporary hide (so, not to discard / destroy things that wrap under it; in the process) for renting some screen estate for the right side of the world. 

So. Cut-off view is perfectly fine; even better, UX-wise; such that user aware of "oh there is something hiding there!" memento! :hehe: 😬

Reply:
But. Sure, blur css trick sound good. One for later, yes!